### PR TITLE
Add verify scripts to cloud-provider-gcp-presubmits.yaml

### PR DESCRIPTION
**Pull request by cici37**
Hold until https://github.com/kubernetes/cloud-provider-gcp/pull/205 merged

### CODE REVIEW

File: config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml [hunk 1]
```diff
@@ -28,9 +28,6 @@ presubmits:
     always_run: true
     decorate: true
     path_alias: k8s.io/cloud-provider-gcp
-    labels:
-      preset-service-account: "true"
-      preset-k8s-ssh: "true"
```
Comment (amwat) [inline config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml:33]: theses presets are needed for the end to end job

Comment (cici37) [reply config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml]: Updated. Thanks!

**Comment by jpbetz** [comment]
Looks good to me.

/lgtm
/approve

/hold until https://github.com/kubernetes/cloud-provider-gcp/pull/205

**Pull request merged**